### PR TITLE
[protobuf] Update protobufs with patch from circleci.

### PR DESCRIPTION
**Pull request by surfnerd**
Used the patch from https://circleci.com/gh/Unity-Technologies/ml-agents/2577#artifacts/containers/0 to regenerate protobuf objects.  Tested in unity with the 3dball scene (training and inference).

### CODE REVIEW

File: UnitySDK/Assets/ML-Agents/Scripts/CommunicatorObjects/UnityRlOutput.cs [hunk 1]
```diff
@@ -27,18 +27,18 @@ public static partial class UnityRlOutputReflection {
...
+            "RW50cnkaSQoSTGlzdEFnZW50SW5mb1Byb3RvEjMKBXZhbHVlGAEgAygLMiQu",
+            "Y29tbXVuaWNhdG9yX29iamVjdHMuQWdlbnRJbmZvUHJvdG8aaQoPQWdlbnRJ",
+            "bmZvc0VudHJ5EgsKA2tleRgBIAEoCRJFCgV2YWx1ZRgCIAEoCzI2LmNvbW11",
+            "bmljYXRvcl9vYmplY3RzLlVuaXR5UkxPdXRwdXQuTGlzdEFnZW50SW5mb1By",
+            "b3RvOgI4AUoECAEQAkIfqgIcTUxBZ2VudHMuQ29tbXVuaWNhdG9yT2JqZWN0",
+            "c2IGcHJvdG8z"));
       descriptor = pbr::FileDescriptor.FromGeneratedCode(descriptorData,
           new pbr::FileDescriptor[] { global::MLAgents.CommunicatorObjects.AgentInfoProtoReflection.Descriptor, },
           new pbr::GeneratedClrTypeInfo(null, new pbr::GeneratedClrTypeInfo[] {
-            new pbr::GeneratedClrTypeInfo(typeof(global::MLAgents.CommunicatorObjects.UnityRLOutput), global::MLAgents.CommunicatorObjects.UnityRLOutput.Parser, new[]{ "GlobalDone", "AgentInfos" }, null, null, new pbr::GeneratedClrTypeInfo[] { new pbr::GeneratedClrTypeInfo(typeof(global::MLAgents.CommunicatorObjects.UnityRLOutput.Types.ListAgentInfoProto), global::MLAgents.CommunicatorObjects.UnityRLOutput.Types.ListAgentInfoProto.Parser, new[]{ "Value" }, null, null, null),
```
Comment (chriselion) [inline UnitySDK/Assets/ML-Agents/Scripts/CommunicatorObjects/UnityRlOutput.cs]: This is expected - the field was removed (and marked reserved) here: https://github.com/Unity-Technologies/ml-agents/pull/2519

**Comment by chriselion** [comment]
For easier diffs, here's the same PR with `black` applied: https://github.com/Unity-Technologies/ml-agents/pull/2619/files

**Comment by chriselion** [comment]
@harperj Yeah, the CI part is happening in https://github.com/Unity-Technologies/ml-agents/pull/2617

File: UnitySDK/Assets/ML-Agents/Scripts/CommunicatorObjects/UnityRlOutput.cs (hunk 1 above)
Comment (surfnerd) [reply UnitySDK/Assets/ML-Agents/Scripts/CommunicatorObjects/UnityRlOutput.cs]: I'm guessing yes, since it's generated.

**Pull request merged**